FacesofLondon/app/src/main/java/com/novoda/wallpaper/droidcon/TimeOfDayCalculator.java

```diff
@@ -1,12 +1,18 @@
 package com.novoda.wallpaper.droidcon;
 
-import java.util.Calendar;
-
 class TimeOfDayCalculator {
```

Comment:
I wouldn't want the `TimeOfDay` itself to be dependent on `Calendar`. 

I find it perfectly fine for the `TimeOfDay` to know which parts of the day include which hours of the day, but it's the calendar that knows what time it is.